business/jaeger_helper.go

```diff
@@ -109,11 +109,14 @@ func GetServices() (services JaegerServices, err error) {
 	return services, err
 }
 
-func contains(a []string, x string) bool {
```

Comment:
This one was added in https://github.com/kiali/kiali/pull/748, but unused since its introduction!!!
I'll remove.